src/applications/personalization/profile/components/notification-settings/NotificationSettings.jsx

```diff
@@ -99,15 +146,11 @@ const NotificationSettings = ({
             // we handle the health care group a little differently
             // TODO: I don't like this check. what does `group3` even mean?
             if (groupId === 'group3') {
-              if (!isPatient) {
-                return null;
```

Comment:
logic for showing/hiding the health care group has been moved to the redux selectors. and we have existing Cypress tests to ensure that the group is removed for non-patients.